charts/operator-wandb/charts/app/templates/_deployment.tpl

```diff
@@ -254,6 +254,12 @@ spec:
                   },
                   "addr": "kafka://$(KAFKA_CLIENT_USER):$(KAFKA_CLIENT_PASSWORD)@$(KAFKA_BROKER_HOST):$(KAFKA_BROKER_PORT)/$(KAFKA_TOPIC_RUN_UPDATE_SHADOW_QUEUE)?producer_batch_bytes=1048576&num_partitions=$(KAFKA_RUN_UPDATE_SHADOW_QUEUE_NUM_PARTITIONS)&replication_factor=3"
                 }
+            - name: GORILLA_ARTIFACTS_GC_BATCH_SIZE
+              value: {{ .Values.global.artifactsGc.BatchSize }}
```

Comment:
Do these values need to be available globally or can we scope these to the application itself `{{ .Values.app.artifactsGc.BatchSize }}` and also move them in the values.yaml

Reply:
yeah, the difference is that if in .Values.app only the app chart (which is where glue runs) will see those values, but in global they are passed to all the charts.

Reply:
Ah, i see. Makes sense, thank you!

Reply:
I moved the values to the `app` values file instead: https://github.com/wandb/helm-charts/blob/main/charts/operator-wandb/charts/app/values.yaml

and updated the vars to reference {{ .Values.artifactsGc.BatchSize }} 

Is this right?

Reply:
Yes, that is correct, i had a mistake in my previous comment, but you got the reference correct